src/github.com/getlantern/flashlight/genconfig/genconfig.go

```diff
@@ -69,6 +75,7 @@ func main() {
 	runtime.GOMAXPROCS(numcores)
 
 	loadDomains()
+	loadWhitelists()
```

Comment:
Okay, that's fine. I'm not attached to the whitelist name and agree that's more specific. How about just proxiedsites instead of proxiedsiteslist for the sake of brevity?